frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/create-local-volume-set-step.tsx

```diff
@@ -108,100 +150,120 @@ export const CreateLocalVolumeSet: React.FC<CreateLocalVolumeSetProps> = ({
-            When installation is finished come back to OpenShift Container Storage to create a
-            StorageCluster.
-            <div className="ceph-ocs-install__lso-alert__button">
-              <Button type="button" variant="primary" onClick={goToLSOInstallationPage}>
-                Install
-              </Button>
-            </div>
-          </Trans>
-        </Alert>
-      ) : (
-        <>
-          <Grid className="ceph-ocs-install__form-wrapper">
-            <GridItem lg={10} md={12} sm={12}>
-              <Form noValidate={false}>
-                <LocalVolumeSetBody
-                  state={state}
-                  dispatch={dispatch}
-                  diskModeOptions={diskModeDropdownItems}
-                  allNodesHelpTxt={allNodesSelectorTxt}
-                  lvsNameHelpTxt={lvsNameSelectorTxt}
-                  taintsFilter={hasOCSTaint}
-                  storageClassName={storageClass.name}
-                />
-              </Form>
-            </GridItem>
-            <GridItem
-              lg={2}
-              lgOffset={10}
-              md={4}
-              mdOffset={4}
-              sm={4}
-              smOffset={4}
-              className="ceph-ocs-install__donut-chart"
-            >
-              <SelectedCapacity dispatch={dispatch} state={state} ns={ns} />
-            </GridItem>
-          </Grid>
-          <ConfirmationModal
-            ns={ns}
-            lvsNodes={lvsNodes}
-            state={state}
-            dispatch={dispatch}
-            setInProgress={setInProgress}
-            setErrorMessage={setError}
-            storageClassName={storageClass.name}
-          />
-          {state.chartNodes.size < MINIMUM_NODES && (
-            <Alert
-              className="co-alert ceph-ocs-install__wizard-alert"
-              variant="danger"
-              title={t('ceph-storage-plugin~Minimum Node Requirement')}
-              isInline
-            >
-              {t(
-                'ceph-storage-plugin~A minimum of 3 nodes are required for the initial deployment. Only {{nodes}} node match to the selected filters. Please adjust the filters to include more nodes.',
-                { nodes: state.chartNodes.size },
-              )}
-            </Alert>
-          )}
-          <RequestErrors errorMessage={error?.message} inProgress={inProgress} />
-        </>
-      )}
+    <ErrorHandler
+      loaded={discoveriesLoaded}
+      loadingMessage={
+        !csvLoaded
+          ? t('ceph-storage-plugin~Checking LSO operator installation')
+          : !discoveriesLoaded
+          ? t('ceph-storage-plugin~Discovering disks on all hosts. This may take a few minutes.')
+          : null
+      }
+      error={discoveriesLoadError}
+      errorMessage={csvLoadError || csv?.status?.phase !== 'Succeeded' ? <LSOInstallAlert /> : null}
+    >
+      <>
+        <Grid>
+          <GridItem lg={8} md={8} sm={8}>
```

Comment:
```suggestion
          <GridItem span={8}>
```
If all values are the same then we can directly use `span`.